<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatWindow</class>
 <widget class="QWidget" name="GroupChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#GroupChatWindow{background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:0.503, y2:0.306727, stop:0.0696517 rgba(169, 169, 169, 255), stop:0.890547 rgba(255, 255, 255, 255));}
#stackedWidget{ background-color: rgba(1,1,1,0);}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="GCAddUsersBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="GCInviteUsersButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>260</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GCBackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GCClearButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="GCTitle">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Group Chat</string>
     </property>
    </widget>
    <widget class="QLabel" name="TopicLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Topic:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TopicBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="AddUsersLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Users:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FirstModBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="FirstModLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;First &lt;br/&gt;Moderator:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
